api/deploy/kubernetes/nfs/csidriver.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: "{{ .Name }}"
+spec:
+  attachRequired: false
+  volumeLifecycleModes:
+    - Persistent
+    - Ephemeral
```

Comment:
attachRequired is false and do we plan to support Ephemeral also?

Reply:
This is the default configuration from the csi-driver-nfs (NodePlugin)... Not sure if there are any special things needed for Ephemeral support?

Reply:
I think For Ephemeral the CreateVolue needs to be taken care of in NodePublish https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html#implementing-csi-ephemeral-inline-support.